pipeline-runner/R/qc-6-integrate_scdata.R

```diff
@@ -318,8 +379,14 @@ run_unisample <- function(scdata, config) {
   return(scdata)
 }
 
-add_dispersions <- function(scdata) {
-  vars <- Seurat::HVFInfo(object = scdata, assay = "RNA", selection.method = "vst")
+add_dispersions <- function(scdata, method) {
+  if (method == "SCT" & Seurat::DefaultAssay(scdata) == "integrated") {
```

Comment:
&& it's the recommended operator for guards 
```suggestion
  if (method == "SCT" && Seurat::DefaultAssay(scdata) == "integrated") {
```